web/src/user/settings/auth/UserSettingsPermissionsPage.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect } from 'react'
+import * as GQL from '../../../../../shared/src/graphql/schema'
+import { PageTitle } from '../../../components/PageTitle'
+import { Timestamp } from '../../../components/time/Timestamp'
+import { eventLogger } from '../../../tracking/eventLogger'
+
+/**
+ * The user settings permissions page.
+ */
+export const UserSettingsPermissionsPage: React.FunctionComponent<{ user: GQL.IUser }> = ({ user }) => {
+    useEffect(() => eventLogger.logViewEvent('UserSettingsPermissions'))
+
+    return (
+        <div className="user-settings-permissions-page w-100">
```

Comment:
```suggestion
        <div className="w-100">
```

Don't think that's used, right?

Reply:
Hmm, yes itself is unused, but I think we have it for every page so we can scope SCSS only to be effective on this page. For example: https://sourcegraph.com/github.com/sourcegraph/sourcegraph@jc/user-perms-ui/-/blob/web/src/repo/settings/RepoSettingsOptionsPage.tsx#L67:57

Not really familiar how whole CSS thing work in React, maybe I'm wrong and it is unnecessary.

Reply:
You only need to add a CSS class if it's used :)